packages/delisp-core/src/compiler.ts

```diff
@@ -116,24 +117,22 @@ function compileTopLevel(syntax: Syntax, env: Environment): JSAST {
   }
 }
 
-export function compileModule(syntax: Syntax): JSAST {
+export function compileModule(syntaxes: Syntax[]): JSAST {
   return {
     type: "File",
     program: {
       type: "Program",
       sourceType: "module",
-      body: [
-        {
-          type: "ExpressionStatement",
-          expression: compileTopLevel(syntax, {})
-        }
-      ]
+      body: syntaxes.map(syntax => ({
+        type: "ExpressionStatement",
+        expression: compileTopLevel(syntax, {})
+      }))
     }
   };
 }
 
-export function compileToString(syntax: Syntax): string {
-  const ast = compileModule(syntax);
+export function compileToString(syntax: Syntax | Module): string {
```

Comment:
This `Syntax | Module` doesn't sound like a good idea.  Should we have 2 functions instead?

It's true we are always going to have those 2 exposed versions of the functions,  one for module and one for syntax.  But should be fine. If not it is better to not expose one returning string, just return JSAST and let the user (index.ts?)  define them.

Reply:
I think returning the JSAST is not a great idea, since the format of that is pretty tied to Recast (which might be a temporary dependency), and I feel this module should simply output JS.

I split this up in 2 functions, where the original one wraps the syntax in a `Module` (`compileModule` itself is not exposed from this file, so would make sense to reuse it -- at least for now).